test/unit/core/AnchorUnit.tree

```diff
@@ -0,0 +1,28 @@
+Anchor::constructor
+├── it should set registry
+└── it should set profileId
+
+Anchor::execute
+├── when caller is not the profileId owner
+│   └── it should revert
+├── when _target is the zero address
```

Comment:
Kind of a nitpick but we should avoid using `_` in when conditions according to the [docs](https://github.com/defi-wonderland/wonder-tests?tab=readme-ov-file#213-coverage): _There should be no underscore in the Given/When conditions (for clarity's sake once using Bulloak for scaffolding)_

Reply:
fixed